src/test/java/example/Example.java

```diff
@@ -21,19 +22,23 @@
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 
+/**
+ * Example applications for ETE testing.
+ */
+@SuppressWarnings("checkstyle:designforextension")
```

Comment:
yea, because i have classes inside that aren't final AND not abstract. it helps subclass the Web example for the REST example... open to there being a cleaner way, but this should be sufficient